### PR TITLE
Edge case tests

### PR DESCRIPTION
**Pull request by tarakpurohit**

### CODE REVIEW

File: test/jira/multiple-jira.test.ts [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+			jiraHost: jira2Host,
+			jiraClientKey: clientKey
+		});
+	});
+
+	afterEach(async () => {
+		await Subscription.destroy({ truncate: true });
+		await Subscription.destroy({ truncate: true });
+		await Installation.destroy({ truncate: true });
+		await Installation.destroy({ truncate: true });
```
Comment (mboudreau) [inline test/jira/multiple-jira.test.ts:39]: You only need 1 of each.

File: test/setup/setup.ts [hunk 2]
```diff
@@ -19,13 +19,17 @@ function resetEnvVars() {
 
 declare global {
 	let jiraHost: string;
+	let jira2Host: string;
```
Comment (mboudreau) [inline test/setup/setup.ts:22]: Please don't put this here as it's only specific to one test suite.  Please move it to the test suite file

File: test/setup/setup.ts [hunk 3]
```diff
@@ -34,7 +38,9 @@ declare global {
 beforeEach(() => {
 	resetEnvVars();
 	global.jiraHost = process.env.ATLASSIAN_URL || "";
+	global.jira2Host = process.env.ALTERNATE_ATLASSIAN_URL || "https://test2-atlassian-instance.net";
```
Comment (mboudreau) [inline test/setup/setup.ts:41]: Dont' use an env var for that, just set one in the test file.

**Review by mboudreau — CHANGES_REQUESTED** [review]
Great test to have.  A few comments to address.  Also, I'd probably place a lot of the nock JSON into it's own separate JSON file as it's kinda hard to read the tests with so many.

**Review by mboudreau — APPROVED** [review]
Much better :+1:

**Review by bgvozdev — APPROVED** [review]
